app/controllers/terms_controller.rb

```diff
@@ -35,4 +36,11 @@ def render_404
   def vocabulary
     @vocabulary ||= Vocabulary.find(params[:vocabulary_id])
   end
+
+  def authorize
```

Comment:
Probably the simplest way is to move it to ApplicationController, which all the controllers inherit from.